apps/testingaccessibility/src/pages/api/progress/index.ts

```diff
@@ -13,7 +13,10 @@ const getProgress = async (req: NextApiRequest, res: NextApiResponse) => {
     req,
     res,
   })
-  const sessionToken = await getDecodedToken(req)
+  const sessionToken = await getToken({
+    req,
+    secret: process.env.NEXTAUTH_SECRET,
+  })
```

Comment:
having a challenging time with the typing of the decoded token

looking at the code for next-auth, this would actually be the default behavior anyway, but didn't test it.